webpack.config.js

```diff
@@ -1,48 +1,38 @@
 /* eslint-disable import/no-extraneous-dependencies */
-const webpack = require('webpack');
```

Comment:
I'd prefer you not change `'` to `"` :( only change what is really necessary, and has a semantic meaning (not pure codestyle)